common/persistence/versionhistory/version_history.go

```diff
@@ -176,6 +176,19 @@ func IsVersionHistoryItemsInSameBranch(versionHistoryItemsA []*historyspb.Versio
 	return lowestCommonAncestor.Equal(aLastItem) || lowestCommonAncestor.Equal(bLastItem)
 }
 
+func SplitVersionHistoryByLastLocalGeneratedItem(
+	versionHistoryItems []*historyspb.VersionHistoryItem,
+	initialFailoverVersion int64,
+	failoverVersionIncrement int64,
+) ([]*historyspb.VersionHistoryItem, []*historyspb.VersionHistoryItem) {
```

Comment:
nit: add the para name to better understand what are the return items?

Reply:
Sure. Will do.